unittests/Packages/Longtable.py

```diff
@@ -13,7 +13,7 @@ def Soup(source):
 except ImportError:
     from BeautifulSoup import BeautifulSoup as Soup
 
-
+@unittest.skip("skip class for now as it uses the binary")
```

Comment:
This needs more attention. My push request is just a starting point. Did not look into test code yet.